src/analysis/special_functions/trigonometric/angle.lean

```diff
@@ -270,6 +271,41 @@ cos_antiperiodic θ
 @[simp] lemma cos_sub_pi (θ : angle) : cos (θ - π) = -cos θ :=
 cos_antiperiodic.sub_eq θ
 
+/-- The sign of a `real.angle` is `0` if the angle is `0` or `π`, `1` if the angle is strictly
+between `0` and `π` and `-1` is the angle is strictly between `-π` and `0`. It is defined as the
+sign of the sine of the angle. -/
+def sign (θ : angle) : sign_type := _root_.sign (sin θ)
```

Comment:
I don't think the `_root_` annotations here and elsewhere are necessary.

Reply:
One use of `_root_` is needed; I've removed the others.